src/extension.ts

```diff
@@ -97,14 +97,8 @@ export function activate(context: vscode.ExtensionContext) {
                 vscode.commands.executeCommand('appService.Refresh', node.getParentNode());
             } catch (err) {
                 if (!(err instanceof UserCancelledError)) {
```

Comment:
Just remove the try/catch completely. Even UserCancelledError should bubble up. `initAsyncCommand` will list the result as 'Canceled' in our telemetry that way